libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/reader/RtpH263Reader.java

```diff
@@ -90,24 +91,25 @@ public void consume(ParsableByteArray data, long timestamp, int sequenceNumber,
     //    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
     int currPosition = data.getPosition();
     int header = data.readUnsignedShort();
-    boolean pBit = ((header & 0x400) == 0x400);
+    boolean pBitIsSet = ((header & 0x400) > 0);
 
-    // Check if optional Video Redundancy Coding or PLEN or PEBIT is present, RFC4629 Section 5.1.
+    // Check if optional V (Video Redundancy Coding), PLEN or PEBIT is present, RFC4629 Section 5.1.
     if ((header & 0x200) != 0 || (header & 0x1f8) != 0 || (header & 0x7) != 0) {
       Log.w(TAG, "Packet discarded due to (VRC != 0) or (PLEN != 0) or (PEBIT != 0)");
       return;
     }
-    int startCodePayload = data.peekUnsignedByte() & 0xfc;
-    if (pBit == true) {
-      if (startCodePayload < 128) {
+
+    if (pBitIsSet == true) {
+      int startCodePayload = data.peekUnsignedByte() & 0xfc;
+      // Packets that begin with a Picture Start Code(100000). Refer RFC4629 Section 6.1.1.
```

Comment:
i have difficulty understanding here: if picture start code is `0b100000` which is 64, then it'll always fail the check `startCodePayload < PICTURE_START_CODE`. Maybe you quoted a wrong RFC section?

Reply:
Picture start code is "100000" from msb, so it is 128. If startCodePayload is less than 128 then packet is follow-On.